priv/http_api/backend_swagger.yaml

```diff
@@ -0,0 +1,430 @@
+# Example YAML to get you started quickly.
+# Be aware that YAML has indentation based scoping.
+# Code completion support is available so start typing for available options.
+swagger: '2.0'
+
+# This is your document metadata
+info:
+  version: "0.0.0"
+  title: MongooseIM control backend REST API
+  description: This REST API provides similar control functionaly to the `mongooseimctl` command. The response objects are modeled on the `ejabberd_commands` entries and print-outs from `mongooseimctl` though the final API will need to be consistent with the MongooseIM's **command registry**.
+schemes:
+  - http
+basePath: /api
+produces:
+  - application/json
+paths:
+  /status:
+    get:
+      description: The status of MongooseIM on the given host.
+      responses:
+        200:
+          description: Success response with the status of the MongooseIM node
+          schema:
+            title: MIMStatus
+            type: object
+            properties:
+              hostname:
+                type: string
+              osPID:
+                type: integer
+              version:
+                type: string
+              uptime:
+                type: string
+              logLocation:
+                type: string
+                format: directory
+  /{XMPPHost}:
+    get:
+      description: Information about the given XMPP host (**what information should we return?**).
+      parameters:
+        - name: XMPPHost
+          in: path
+          description: The XMPP hostname
+          required: true
+          type: string
+          format: hostname
+      responses:
+        200:
+          description: Successful response with information about the given XMPP host
+          schema:
+            title: XMPPHost
+            type: object
+            properties:
+              name:
+                type: string
+              information:
+                type: string
+        404:
+          description: Failure response because the given XMPP host does not exist
+  /{XMPPHost}/roster:
```

Comment:
I don't know if we need that method. More important (IMO) is to get list of all registered users. Heaving that we can query the API for specific user's roster.